projects/js-packages/scan/src/types/fixers.ts

```diff
@@ -0,0 +1,17 @@
+export type FixerStatus = 'not_started' | 'in_progress' | 'fixed' | 'not_fixed';
```

Comment:
Just a general note on these types, in Protect we have a much more extensive [set of types](https://opengrok.a8c.com/source/xref/jetpack/projects/plugins/protect/src/js/types/fixers.ts?r=fdcd55ea) in `fixers.ts` that considers all the varying possible statuses. It seems we don't require this to function here but will that change once we are handling actual Scan API responses? Or are these two just similar but for completely separate purposes and wont have any overlap?

Reply:
I've only moved over the necessary types used by the dataviews component in this PR – the component doesn't work with the fixers API response directly, it only needs the `FixerStatus` and `ThreatFixStatus` types for rendering the auto-fix column. 

I do eventually bring in the rest of the `FixersStatus` types [here](https://github.com/Automattic/jetpack/pull/39767/files#diff-ff61780beddfa13c57a1dfa37296bd1ed4a823f9145328761412c9dda7bfe27f) in the [follow-up PR](https://github.com/Automattic/jetpack/pull/39767) :+1: